frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import { TracimComponent } from '../../tracimComponent.js'
+import { appContentFactory } from '../../appContentFactory.js'
+import { tinymceRemove } from '../../helper.js'
+import CardPopup from '../CardPopup/CardPopup.jsx'
+import IconButton from '../Button/IconButton.jsx'
+
+const wysiwygId = 'wysiwygTextToToDos'
+const wysiwygIdSelector = `#${wysiwygId}`
+
+const CreateToDoFromTextPopup = (props) => {
+  useEffect(() => {
+    return tinymceRemove(wysiwygIdSelector)
+  }, [])
+
+  return (
+    <CardPopup
+      customClass='createToDoFromTextPopup'
+      customColor={props.customColor}
+      onClose={props.onClickClose}
+      label={props.t('Transorm todos')}
+      faIcon='fas fa-edit'
+    >
+      <div className='createToDoFromTextPopup__main'>
+        <span>
+          {props.t('Enter your tasks bellow:')}
+        </span>
+        <textarea
+          placeholder={props.placeHolder || props.t('Your message...')}
+          onChange={props.onChangeValue}
+          value={props.value}
+        />
+      </div>
+      <div className='createToDoFromTextPopup__buttons'>
+        <IconButton
+          color={props.customColor}
+          icon='fas fa-times'
+          intent='secondary'
+          mode='dark'
+          onClick={props.onClickClose}
+          text={props.t('Cancel')}
+          type='button'
+        />
+        <IconButton
+          color={props.customColor}
+          disabled={false}
+          icon='far fa-paper-plane'
+          intent='primary'
+          mode='light'
+          onClick={props.onClickTransform}
+          text={props.t('Create task from text')}
+          type='button'
+        />
+      </div>
+    </CardPopup>
+  )
+}
+
+export default translate()(appContentFactory(TracimComponent(CreateToDoFromTextPopup)))
```

Comment:
Do you need appContentFactory and TracimComponent?

Reply:
Indeed, I don't need them
Removed in 2029211